assets/README.md

```diff
@@ -31,6 +31,7 @@ themes (`bat cache --clear`).
 
 The following files have been manually modified after converting from a `.tmLanguage` file:
 
+* `Apache.sublime_syntax`=> removed `.conf` and `.CONF` file types.
```

Comment:
Do we still need this if we have the `"*.conf", MappingTarget::MapToUnknown` mapping?

Reply:
Strangely, the test `no_duplicate_extensions` fails when the lines are not removed from the Apache syntax file. Should I add an entry to `KNOWN_EXCEPTIONS` in this test?